.travis.yml

```diff
@@ -1,18 +1,23 @@
-dist: xenial
-language: python
-python:
-  - 3.6
+language: generic
+env:
+    matrix:
+        - TEST_TYPE=integ DOCKER_IMAGE=nmstate/centos7-nmstate-dev
+        - TEST_TYPE=integ DOCKER_IMAGE=nmstate/fedora-nmstate-dev
```

Comment:
Does each combination runs on its own VM or on the same one? How much time it takes now compared to before?

Reply:
Each has its own VM.